variable/technology/power-plant.yaml

```
@@ -0,0 +1,799 @@
+                plant category.
+   unit: GW/h
+
+Maximum Ramping|Upwards|Electricity|Oil:
+   description: Upward ramp limit of the power generation unit who belongs to the specified power
+                plant category.
+   unit: GW/h
+
+Maximum Ramping|Downwards|Electricity|Coal:
+   description: Downward ramp limit of the power generation unit who belongs to the specified power
+                plant category.
+   unit: GW/h
+
+Maximum Ramping|Downwards|Electricity|Gas:
+   description: Downward ramp limit of the power generation unit who belongs to the specified power
+                plant category.
+   unit: GW/h
+
+Maximum Ramping|Downwards|Electricity|Nuclear:
+   description: Downward ramp limit of the power generation unit who belongs to the specified power
+                plant category.
+   unit: GW/h
+
+Maximum Ramping|Downwards|Electricity|Oil:
+   description: Downward ramp limit of the power generation unit who belongs to the specified power
+                plant category.
+   unit: GW/h
+
+# description of simple hydrosystems - reservoir hydro
+# this describes systems composed of one reservoir and one plant (with potential pumping capacity) that are used for seasonal storage
+Output Energy of the Second Block|Electricity|Hydro:
+   description: Generator output of the second block (i.e., above the minimum load)
+   unit: GW
+
+Number of Units|Electricity|Hydro|Reservoir:
+   description: Number of units of kind "simple Hydro reservoir "
+   unit:
+
+Maximum Active power|Electricity|Hydro|reservoir:
+   description: Maximum active power of one typical "simple Hydro reservoir " unit
+   unit: MW
+
+Minimum Active power|Electricity|Hydro|Reservoir:
+   description: Minimum active power of one typical "simple Hydro reservoir " unit
+   unit: MW
+
+Maximum Storage|Electricity|Hydro|reservoir:
+   description: Maximum volume of a reservoir expressed in energy
+   unit: MWh
+
+Minimum Storage|Electricity|Hydro|Reservoir:
+   description: Minimum volume of a reservoir expressed in energy
+   unit: MWh
+
+Pumping Efficiency|Electricity|Hydro|Reservoir:
+   description: efficiency of pumping for a reservoir power plant
+   unit: %
+
+Inertia|Electricity|Hydro|Reservoir:
+   description: inertia provided to the system by a typical "simple Hydro reservoir " unit
+   unit: second
+
+Inflows|Electricity|Hydro|Reservoir:
+   description: hydraulic inflows to a reservoir expressed in energy
+   unit: MWh
+
+Inflows|Electricity|Hydro|Reservoir|Profile:
+   description: Time serie of hydraulic inflows to a reservoir expressed in energy
+   unit: MWh
+
+# description of simple hydrosystems -  pumped storage
+# this describes systems composed of one small reservoir and one plant with  pumping capacity that are used for short-term storage (eg week or day)
+Number of Units|Electricity|Hydro|Pumped Storage:
+   description: Number of units of kind "Pumped Storage "
+   unit:
+
+Maximum Active power|Electricity|Hydro|Pumped Storage:
```

please change throughout
```suggestion
Maximum Active Power|Electricity|Hydro|Pumped Storage:
```